lib/metadata/index.js

```diff
@@ -98,24 +99,26 @@ Metadata.prototype.buildTokens = function (hosts) {
   if (!this.tokenizer) {
     return this.log('error', 'Tokenizer could not be determined');
   }
+  const self = this;
```

Comment:
Just a suggestion, you can use arrow functions below and avoid the need of a `self` field.